src/command_modules/azure-cli-eventgrid/azure/cli/command_modules/eventgrid/_help.py

```diff
@@ -89,7 +89,7 @@
           long-summary: |
             Example: --deadletter-endpoint /subscriptions/{SubID}/resourceGroups/rg1/providers/Microsoft.Storage/storageAccounts/sa1/blobServices/default/containers/containerName
         - name: --endpoint-type
-          short-summary: The type of the destination endpoint.
+          short-summary: The type of the destination endpoint. It is expected that the destination endpoint be created and available for use before executing any Event Grid command.
```

Comment:
The added line should be the long-summary.

Reply:
thx for the feedback. fixed.